CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.10.0
+
+- Add support for Snapdragon 855 CPU
```

Comment:
Maybe `SoC` is better than `CPU`? Or just `Snapdragon 855`

Reply:
Lets go with plain `Snapdragon 855`